### DISCUSSION TITLE
Is there a way to use Bindings with Controllers with Constructor parameters?

### DISCUSSION THREAD

**Discussion by oliverbytes** [jonataslaw/getx#1124] [Q&A] [answered]
Basically, I would like to pass parameters during bindings dependency initialization

```
class FeedsTileBinding implements Bindings {
  @override
  void dependencies() {
    Get.lazyPut<Controller>(() => Controller(<param-passed-from-constructor>));
  }
}
```

Also for passing constructor parameters using GetView?

**Comment by eduardoflorence** [comment, marked as answer]
Yes, it is possible.
See: https://github.com/jonataslaw/getx/blob/master/example/lib/pages/home/bindings/home_binding.dart

**Reply by oliverbytes** [reply]
> Yes, it is possible.
> See: https://github.com/jonataslaw/getx/blob/master/example/lib/pages/home/bindings/home_binding.dart

Thanks so much
